### ISSUE TITLE
Game state reprezentáció

### ISSUE THREAD

**Issue by lipk** [lipk/pyzertz#5]
Ki kell találni egy olyan adatstruktúrát, amivel reprezentálható a játék állása egy adott pillanatban (nyilván elsősorban a tábláról van szó), kényelmesen & gyorsan elvégezhetők rajta a tipikus műveletek (golyó elhelyezése, ugrás stb.).

Azon is lehet gondolkodni, hogy az AI-nak milyen infókra lehet majd szüksége, és ezek hatékony végrehathatóságát is figyelembe venni, de ezt nyilván nem tudjuk 100%-ig előre kitalálni.

**Comment by bgd54** [comment]
A játékosok állapota is hasznos lehet nem? Vagy az lehet még későbbi lépés.

**Comment by lipk** [comment]
Ja, de az egyszerű. A trükkös rész itt a hatszögletű tábla lesz, azon kell leginkább gondolkozni :)

**Comment by sikba** [comment]
OK
